packages/ai-jsx/src/react/index.ts

```diff
@@ -85,6 +87,12 @@ export interface UseAIStreamOpts {
    * An event handler that runs when the request fails.
    */
   onError?: (error: Error) => void;
+
+  /**
+   * A map of component names to React components. This is used to deserialize
+   * the stream.
```

Comment:
This is a user-facing type, right? Let's add a bit more detail / an example here. Imagine the user doesn't know anything about AI.JSX.